<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Holger Mairon</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>-  DKV2 -</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="EmptyPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;H2&gt;Willkommen zu DKV2- Deiner Verwaltung von Direktrediten&lt;/H2&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PersonsListPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableView" name="PersonsTableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="toolTip">
           <string>Doppelklick oder F2 um ein Feld zu ändern. Rechte Maustaste für weitere Optionen</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NewPerson">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="hlNeuePersonButtons">
          <item>
           <widget class="QPushButton" name="saveNew">
            <property name="toolTip">
             <string>Der Datensatz wird gespeichert und ein weiterer Datensatz angelegt</string>
            </property>
            <property name="text">
             <string>Speichern und neu</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveList">
            <property name="toolTip">
             <string>Der Datensatz wird gespeichert und die Liste der DK Geber wird angezeigt</string>
            </property>
            <property name="text">
             <string>Speichern und zur Liste</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveExit">
            <property name="toolTip">
             <string>Der Datensatz wird gespeichert und die Eingabe neuer DK Geber wird beendet</string>
            </property>
            <property name="text">
             <string>Speichern und Ende</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancel">
            <property name="toolTip">
             <string>Das Anlegen eines DK Gebers wird abgebrochen</string>
            </property>
            <property name="text">
             <string>Abbrechen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lVorname">
            <property name="text">
             <string>Vorname:</string>
            </property>
            <property name="buddy">
             <cstring>leVorname</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leVorname"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lNachname">
            <property name="text">
             <string>Nachname:</string>
            </property>
            <property name="buddy">
             <cstring>leNachname</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leNachname"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lStrasse">
            <property name="text">
             <string>Straße:</string>
            </property>
            <property name="buddy">
             <cstring>leStrasse</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leStrasse"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lPlzStadt">
            <property name="text">
             <string>PLZ - Stadt</string>
            </property>
            <property name="buddy">
             <cstring>lePlz</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
            <item>
             <widget class="QLineEdit" name="lePlz"/>
            </item>
            <item>
             <widget class="QLineEdit" name="leStadt"/>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lIban">
            <property name="text">
             <string>IBAN:</string>
            </property>
            <property name="buddy">
             <cstring>leIban</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="leIban"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lBic">
            <property name="text">
             <string>BIC:</string>
            </property>
            <property name="buddy">
             <cstring>leBic</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="leBic"/>
          </item>
          <item row="5" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="NewContractPage">
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="vlNeuerVertrag">
        <item>
         <layout class="QHBoxLayout" name="hlNeuerVertragButtons">
          <item>
           <widget class="QPushButton" name="saveContractGoToDKGeberList">
            <property name="toolTip">
             <string>Speichert den Vertrag und kehrt zur Liste der DK Geber zurück</string>
            </property>
            <property name="text">
             <string>Speichern zur Geber Liste</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveContractGoContracts">
            <property name="toolTip">
             <string>Speichert den Vertrag und geht zur Liste der Verträge</string>
            </property>
            <property name="text">
             <string>Speichern zu Verträgen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cancelCreateContract">
            <property name="toolTip">
             <string>Abbrechen und zurück</string>
            </property>
            <property name="text">
             <string>Abbrechen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>DK Geber</string>
            </property>
            <property name="buddy">
             <cstring>cbDKGeber</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbDKGeber"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Kennung</string>
            </property>
            <property name="buddy">
             <cstring>leKennung</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="leKennung"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Betrag (Euro)</string>
            </property>
            <property name="buddy">
             <cstring>leBetrag</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="leBetrag">
            <property name="inputMask">
             <string>900000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Wert incl. Zinsen (Euro)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="lblWert">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Zinssatz</string>
            </property>
            <property name="buddy">
             <cstring>cbZins</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Vertragsabschluss (Datum)</string>
            </property>
            <property name="buddy">
             <cstring>deVertragsabschluss</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Beginn des nächsten Zinszeitraums</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="lblBeginZinsphase">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Laufzeitende (Datum)</string>
            </property>
            <property name="buddy">
             <cstring>deLaufzeitEnde</cstring>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDateEdit" name="deLaufzeitEnde"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Zinsen werden aufgerechnet</string>
            </property>
            <property name="buddy">
             <cstring>chkbTesaurierend</cstring>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QCheckBox" name="chkbTesaurierend">
            <property name="text">
             <string>Tesaurierend</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="cbZins"/>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="deVertragsabschluss"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblHinweis">
          <property name="text">
           <string>&lt;H3&gt; Nach dem Anlegen müssen die Verträge nach Zahlungseingang aktiviert werden, damit die Zinsberechnung beginnt!</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ContractsListPage">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTableView" name="contractsTableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatenbank">
    <property name="title">
     <string>DK Datenbank</string>
    </property>
    <addaction name="action_Neue_DB_anlegen"/>
    <addaction name="actionDBoeffnen"/>
    <addaction name="actionDB_schliessen"/>
    <addaction name="separator"/>
    <addaction name="actionProgramm_beenden"/>
   </widget>
   <widget class="QMenu" name="menuDK_Geber">
    <property name="title">
     <string>DK Geber</string>
    </property>
    <addaction name="action_Liste"/>
    <addaction name="separator"/>
    <addaction name="actionNeuer_DK_Geber"/>
    <addaction name="actionKreditgeber_l_schen"/>
   </widget>
   <widget class="QMenu" name="menuVertr_ge">
    <property name="title">
     <string>Verträge</string>
    </property>
    <addaction name="actionListe_der_Vertr_ge_anzeigen"/>
    <addaction name="separator"/>
    <addaction name="actionVertrag_anlegen"/>
    <addaction name="actionVertrag_l_schen"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <addaction name="actioncreateSampleData"/>
    <addaction name="actionanzeigenLog"/>
   </widget>
   <addaction name="menuDatenbank"/>
   <addaction name="menuDK_Geber"/>
   <addaction name="menuVertr_ge"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Neue_DB_anlegen">
   <property name="text">
    <string>&amp;Neue Anlegen</string>
   </property>
   <property name="toolTip">
    <string>Legen Sie hiermit eine leere DK Datenbank an</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDBoeffnen">
   <property name="text">
    <string>&amp;Öffnen</string>
   </property>
   <property name="toolTip">
    <string>Öffnen Sie hiermit eine existierende Datenbank</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDB_schliessen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Schließen</string>
   </property>
   <property name="toolTip">
    <string>Schließen Sie hiermit die geöffnete Datenbank</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_Liste">
   <property name="text">
    <string>&amp;Liste der DK Geber</string>
   </property>
   <property name="toolTip">
    <string>Zeigt eine Liste aller DK Geber an</string>
   </property>
  </action>
  <action name="actionProgramm_beenden">
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="actionListe_der_Vertr_ge_anzeigen">
   <property name="text">
    <string>Liste der Verträge anzeigen</string>
   </property>
  </action>
  <action name="actioncreateSampleData">
   <property name="text">
    <string>Create Sample Data</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNeuer_DK_Geber">
   <property name="text">
    <string>Anlegen</string>
   </property>
  </action>
  <action name="actionVertrag_anlegen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anlegen</string>
   </property>
   <property name="toolTip">
    <string>Vertrag für ausgewählten DK Geber anlegen</string>
   </property>
  </action>
  <action name="actionKreditgeber_l_schen">
   <property name="text">
    <string>Löschen</string>
   </property>
   <property name="toolTip">
    <string>Löscht einen Kreditgeber mit seinen Verträgen </string>
   </property>
  </action>
  <action name="actionshowLog">
   <property name="text">
    <string>showLog</string>
   </property>
  </action>
  <action name="actionactivateContract">
   <property name="text">
    <string>activateContract</string>
   </property>
   <property name="toolTip">
    <string>Zinszahlung des Vertrags aktivieren</string>
   </property>
  </action>
  <action name="actiondeactivateContract">
   <property name="text">
    <string>deactivateContract</string>
   </property>
  </action>
  <action name="actiondeleteContract">
   <property name="text">
    <string>deleteContract</string>
   </property>
  </action>
  <action name="actionVertrag_l_schen">
   <property name="text">
    <string>Löschen</string>
   </property>
  </action>
  <action name="actionanzeigenLog">
   <property name="text">
    <string>Open Log </string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cbDKGeber</tabstop>
  <tabstop>leKennung</tabstop>
  <tabstop>leBetrag</tabstop>
  <tabstop>cbZins</tabstop>
  <tabstop>deVertragsabschluss</tabstop>
  <tabstop>deLaufzeitEnde</tabstop>
  <tabstop>chkbTesaurierend</tabstop>
  <tabstop>saveContractGoToDKGeberList</tabstop>
  <tabstop>saveContractGoContracts</tabstop>
  <tabstop>cancelCreateContract</tabstop>
  <tabstop>leVorname</tabstop>
  <tabstop>leNachname</tabstop>
  <tabstop>leStrasse</tabstop>
  <tabstop>lePlz</tabstop>
  <tabstop>leStadt</tabstop>
  <tabstop>leIban</tabstop>
  <tabstop>leBic</tabstop>
  <tabstop>saveNew</tabstop>
  <tabstop>saveList</tabstop>
  <tabstop>saveExit</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>PersonsTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
